pageserver/src/http/routes.rs

```diff
@@ -1943,6 +1975,10 @@ pub fn make_router(
         .post("/v1/tenant/:tenant_id/ignore", |r| {
             api_handler(r, tenant_ignore_handler)
         })
+        .post(
+            "/v1/tenant/:tenant_shard_id/timeline/:timeline_id/preserve_initdb_archive",
+            |r| api_handler(r, timeline_preserve_initdb_handler),
+        )
```

Comment:
IIUC this API will be used by the DR workflow, so, it's a committed interface. Should document it in our `openapi_spec.yml`

Reply:
Done.